management/client.go

```diff
@@ -229,6 +229,84 @@ type OIDCBackchannelLogout struct {
 	BackChannelLogoutURLs *[]string `json:"backchannel_logout_urls,omitempty"`
 }
 
+// Addons defines the `addons` settings for a Client.
+type Addons struct {
+	// SAML2 Addon configuration. Set this property to indicate that the Addon should be enabled, all settings are optional.
+	// The first entry in `Callbacks` should be the URL to post the SAML Token to
+	SAML2 *SAML2Addon `json:"samlp,omitempty"`
+	// WS-Fed Addon. Set this property to indicate the Addon should be enabled and then store the
+	// configuration in `Callbacks` and `ClientAliases` properties on the Client.
+	// The first entry in `Callbacks` should be the URL to post the SAML Token to.
+	// ClientAliases should include the Realm which is the identifier sent by the application.
+	WSFED *WSFEDAddon `json:"wsfed,omitempty"`
```

Comment:
```suggestion
	WSFED *ClientAddonWSFED `json:"wsfed,omitempty"`
```

Shoul we rename this as well to this? 🤔